### PR TITLE
#161551220 Scaffold the project

### PR DESCRIPTION
**Pull request by kevinsamoei**
#### What does this PR do?
Make a scaffold of the project and all the required initial setups.
#### Description of Task to be completed?
To initialize the base with which the project will be built.
* As a Developer
I want to clone the bp-andela-media repository which has both rails and vue scaffold base code
So that I can be able to build on top of it.
* I have also configured `Rspec` and added `factory_bot_rails` and `simplecov`
* Added and configured environment variables using `figaro`
#### How should this be manually tested?
* Clone the project and test it out for any arrors.
#### What are the relevant pivotal tracker stories?
[#161551220](https://www.pivotaltracker.com/story/show/161551220)
#### Any background context you want to add?
N/A
#### Screenshots
N/A

### CODE REVIEW

**Comment by gr1d99** [comment]
How about setting up these @kevinsamoei 
- [ ] **factory_bot_rails**
- [ ] **RSpec**
- [ ] **Figaro**
- [ ] **Simplecov**

**Comment by kevinsamoei** [comment]
@gr1d99 On it.

**Comment by kevinsamoei** [comment]
@andela/andela-media-developers Kindly review this PR ASAP. It is a blocker to most tasks.

**Comment by kevinsamoei** [comment]
@gr1d99 Done.

**Comment by kevinsamoei** [comment]
@gr1d99 Rubocop added.

**Comment by chalchuck** [comment]
I am suggesting we use the Default Rails Credentials to handle ENV variables instead of using gem Figaro @kevinsamoei what day you?

**Comment by chalchuck** [comment]
Can we please remove all unnecessary comments from the code??

**Comment by chalchuck** [comment]
Let us remove all references to Rails core modules that will not be utilized in this project example ActiveCable.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+/config/master.key
+/public/packs
+/public/packs-test
+/node_modules
+yarn-debug.log*
+.yarn-integrity
+
+# Ignore application configuration
+/config/application.yml
+coverage
```
Comment (kasigazibrian) [inline .gitignore]: Could we add `.ds_store`  to the `.gitignore`. This is usually generated by the Apple macOS operating system. It stores custom attributes of its containing folder.

**Comment by kevinsamoei** [comment]
@kasigazibrian Done
@chalchuck My changes have been made. Waiting for your PR to be up.
@andela/andela-media-developers I think this should be closed. Most of the issues have been ironed out. Other issues shall be fixed by other PR's. Let's ensure this is closed early so that other peeps could start on their task.

**Pull request merged**